contracts/duckies/games/DuckyFamily/DuckyFamilyV1.sol

```diff
@@ -173,6 +173,11 @@ contract DuckyFamilyV1 is IDuckyFamily, AccessControl, Random {
 		maxPeculiarity = _calcMaxPeculiarity();
 	}
 
+	// ------- Random -------
+	function setPepper(bytes32 pepper) external onlyRole(MAINTAINER_ROLE) {
+		_setPepper(pepper);
```

Comment:
But why making a private method if it's only for external use ?

Maybe we call it

Seed = random number set by server
Salt = blockhash + timestamp +++

Reply:
`Random._setPepper()` is private as `Random` does not use `AccessControl`. Thus, Game should control who can call this function itself.

Reply:
My naming logic was the following:
- nonce - number of seed generations.
- salt - data that changes after each seed generation.
- pepper - data set by an external entity periodically
- seed - data, generated from nonce, salt and pepper, that takes part in generating random number